install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,78 @@ Check VNIDs by running:
 $ oc get netnamespace
 ----
 
+[[renumbering-sdn-networks]]
+== Renumbering the SDN Network
+
+It is difficult to change the networking configuration of an
+{product-title} cluster after it has been deployed, but if absolutely
+necessary, it can be done in some cases.
+
+=== Expanding the Cluster IP Range By Adding Nodes
+
+The easiest reconfiguration involves changing the
+`*clusterNetworkCIDR*` to make room to add additional nodes, while
+leaving existing Nodes and Pods untouched. (That is, changing
+`*clusterNetworkCIDR*` to a new value which is larger than the old
+value, and which includes the old value as a subset.)
+
+For example, with the default `*clusterNetworkCIDR*` value of
+*10.128.0.0/14* and the default `*hostSubnetLength*` of *9*, you can
+have 512 nodes, with Pod subnets from *10.128.0.0/23* to
+*10.131.254.0/23*. (Older versions of {product-title} had smaller
+defaults.) If you changed `*clusterNetworkCIDR*` to *10.128.0.0/13*,
+then you could have an additional 512 nodes, with Pod IPs going up to
+*10.135.255.255*.
+
+To do this, simply change the `*clusterNetworkCIDR*` value in the
+`*networkConfig*` stanza of the *_master-config.yaml_* file, and then
+restart the master; for single master clusters:
+
+----
+# systemctl restart atomic-openshift-master
+----
+
+For multi-master clusters, on each master:
+
+----
+# systemctl restart atomic-openshift-master-controllers
+----
+
+You will also need to restart the node service on the master and on
+each of the nodes, to make them update their IP routing information to
+take the larger cluster network into account:
+
+----
+# systemctl restart atomic-openshift-node
+----
+
+As stated above, this only works when the new `*clusterNetworkCIDR*`
+value fully contains the original value; if you try to change it to
+any other value, then when you restart the master, it will complain
+about the mismatch and refuse to start.
+
+=== Other Changes to the Cluster or Service Network IP Range
+
+If you want to completely change `*clusterNetworkCIDR*` (eg, moving
+from *10.128.0.0/14* to *192.168.9.0/8*) or you want to change
+`*hostSubnetLength*` (to allow more or fewer pods per node) or
+`*serviceNetworkCIDR*`, simply changing the configuration and
+restarting will not work.
+
+ifndef::openshift-enterprise[]
+Although it is theoretically possible to make such changes in a
+deployed cluster, it is not supported and not documented here. While
+{product-name} may support such changes in the future, for now the
+best solution is to reinstall the cluster with the new correct values.
+endif::[]
+ifdef::openshift-enterprise[]
+Although it is possible to make such changes in a deployed cluster, it
+is complicated and not fully supported; if it all possible, you should
+consider reinstalling the cluster instead. FIXME talk to your support
+person or look up the answer in the support db or something, I'm not
+sure what to write here...
+endif::[]
```

Comment:
No, not yet... we haven't finished writing it yet. But I guess we can get a placeholder article created now to reserve a URL.